src/padding.rs

```diff
@@ -0,0 +1,99 @@
+use crate::CellOption;
+use papergrid::{Entity, Grid, Settings};
+
+/// Padding is responsible for a left/right/top/bottom indent of particular and fill one set characters.
+///
+/// ```rust,no_run
+///   # use tabled::{Style, Padding, Row, Table, Modify};
+///   # let data: Vec<&'static str> = Vec::new();
+///     let table = Table::new(&data).with(Modify::new(Row(..1)).with(
+///                             Padding::new(0, 0, 1, 1)
+///                                 .set_left_fill('>')
+///                                 .set_right_fill('<')
+///                                 .set_bottom_fill('^')
+///                                 .set_top_fill('V')));
+/// ```
+#[derive(Debug, Clone, Copy)]
+pub struct Padding {
+    pub top: Indent,
+    pub bottom: Indent,
+    pub left: Indent,
+    pub right: Indent,
+}
+
+impl Padding {
+    /// Construct's an Indent object.
+    pub fn new(left: usize, right: usize, top: usize, bottom: usize) -> Self {
+        Self {
+            top: Indent::default().set_size(top),
+            bottom: Indent::default().set_size(bottom),
+            left: Indent::default().set_size(left),
+            right: Indent::default().set_size(right),
+        }
+    }
+
+    pub fn set_top_fill(mut self, fill: char) -> Self {
+        self.top.fill = fill;
+        self
+    }
+
+    pub fn set_bottom_fill(mut self, fill: char) -> Self {
+        self.bottom.fill = fill;
+        self
+    }
+
+    pub fn set_left_fill(mut self, fill: char) -> Self {
+        self.left.fill = fill;
+        self
+    }
+
+    pub fn set_right_fill(mut self, fill: char) -> Self {
+        self.right.fill = fill;
+        self
+    }
+}
+
+impl CellOption for Padding {
+    fn change_cell(&mut self, grid: &mut Grid, row: usize, column: usize) {
+        grid.set(
+            &Entity::Cell(row, column),
+            Settings::new().padding(
+                papergrid::Indent {
+                    fill: self.left.fill,
+                    size: self.left.size,
+                },
+                papergrid::Indent {
+                    fill: self.right.fill,
+                    size: self.right.size,
+                },
+                papergrid::Indent {
+                    fill: self.top.fill,
+                    size: self.top.size,
+                },
+                papergrid::Indent {
+                    fill: self.bottom.fill,
+                    size: self.bottom.size,
+                },
+            ),
+        )
+    }
+}
+
+#[derive(Debug, Clone, Copy)]
+pub struct Indent {
+    pub fill: char,
+    pub size: usize,
+}
```

Comment:
Am I wrong that `Indent` can stay private?

Reply:
it can be removed by the way by using it from `papergrid`.
But look at my previous comment